clients/fides-js/src/fides.ts

```diff
@@ -248,14 +248,12 @@ const init = async ({
   _Fides.options = options;
   _Fides.initialized = true;
 
-  if (getConsentContext().globalPrivacyControl) {
-    effectiveExperience?.privacy_notices?.forEach((notice) => {
-      if (notice.has_gpc_flag) {
-        // todo- write cookie with user preference
-        // todo- send consent request downstream automatically with saveUserPreference()
-      }
-    });
-  }
+  automaticallyApplyGPCPreferences(
+    cookie,
+    fidesRegionString,
+    options.fidesApiUrl,
+    effectiveExperience
+  );
```

Comment:
hmm what should be the behavior of gpc opts the user out of everything? would we not want the banner to show, or perhaps show it but say gpc opted them out of everything? probably a design/product question. but right now if all of my notices are gpc enabled, and then gpc opts me out, the banner still comes up without any indication that I've been opted out (until I open up "Manage preferences" and see the toggles aren't active)

Reply:
oh yes, right now there are no indications of GPC applied, but might make sense for a future iteration